error.md

```diff
@@ -0,0 +1,62 @@
+# Errors faced during migration with nikita v1
+
+## Installation of semver necessary
+
+When running the `bootstrap` command, I got an error stating that `semver` module was not found.
+
+### Solution
+
+npm install semver
+
+## SSH not working but no error
+
+Nikita actions are not being executed. The code exits after entering the input
+
+### Solution
+
+Worked after refactoring the code in `lib/run.coffee`
+
+```coffee
+  nikita
+    .log.cli(config.cli_config)
+    .log.md(config.log_path)
+    .ssh.open(metadata: header: 'SSH open', config.ssh)
+    .ssh.close(metadata: header: 'SSH closed')
+```
+
+## SSH connection not established
+
+Connection cannot be established.
+
+### Solution
+
+The issue was due to improper config object property. The config object passed was the following:
+
+```coffee
+{
+  disabled: false,
+  host: '192.168.1.171',
+  port: '22',
+  password: 'rcarcarca'
+}
+```
+
+I noticed several things.
+
+* First, connection `port` should be an integer to work.
```

Comment:
This is not the case if you have liked nikita with nikita-arch. If you haven't done it, you can not expect to migrate with alpha.0, too many issues

Reply:
What do you mean by "linked" ?

I checked on the documentation and on the schema the port is said to be `integer`.

I am missing something here.